src/components/MovieListItem.vue

```diff
@@ -34,6 +37,10 @@
 import { IonIcon, IonItem, IonLabel, IonNote } from "@ionic/vue";
```

Comment:
you have to add the ion component that you used in this import to make things work. Any ion component in the code above should be in this import.

Reply:
```suggestion
import { IonIcon, IonItem, IonLabel, IonNote, IonButtons } from "@ionic/vue";
```